tests/unit/lms/validation/authentication/_lti_test.py

```diff
@@ -120,6 +120,16 @@ def test_it_raises_missing_application_instance(
         with pytest.raises(ValidationError):
             schema.lti_user()
 
+    def test_it_missing_v13(self, pyramid_request):
+        pyramid_request.lti_jwt = None
```

Comment:
```suggestion
        pyramid_request.lti_jwt = {}
```

I think the jwt method returns empty dict if this isn't set